static/styles/_components/_panel.scss

```diff
@@ -29,11 +43,35 @@
   }
 
   td {
-    border-bottom: 0;
+    border-bottom: 0 !important;
     padding: .3rem 0 !important;
 
     &:first-child {
       font-weight: bold;
     }
   }
+
+  h4 {
+    @include font-size(1.6);
+    font-weight: 600;
+  }
+
+  @include media($medium) {
+    border-bottom: 1px solid $light-gray;
+    padding: 10px 18px;
+    margin-top: 0;
+
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+}
+
+#datatable-modal .panel .panel__row {
+  // TODO this shit is broken.
```

Comment:
Haha, I was supposed to go back to that and write a better comment, I'll do so now